azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -243,6 +220,28 @@ internal class ParticipantListView(
             )
         )
 
+        val plusMoreParticipants = totalActiveParticipantCount - remoteParticipantCellModels.count()
+
+        if (plusMoreParticipants > 0) {
```

Comment:
Lets say, totalParticipant count is 5. 
Myself including all participants are consumers. 

would it show +4 more participants or +5? Since we already excluded myself from the totalParticipant by negating by 1, it should say +4. 
Commenting to re-check for edge cases.

Reply:
I did test, but we will need to have more test cases to test.